coatnet/models/blocks.py

```diff
@@ -22,3 +26,90 @@ def conv_3x3(input_layer, num_filters: int = 64, downsample: bool = False):
     bn_1 = layers.BatchNormalization()(conv_1)
     act_1 = tf.nn.gelu(bn_1)
     return act_1
+
+
+def mbconv_block(
+    input_layer,
+    input_channel: int,
+    output_channel: int,
+    strides: int = 1,
+    expansion_rate: int = 4,
+    se_ratio: float = 0.25,
+    dropout_rate: float = 0.1,
+):
+    """
+    Mobile Inverted Block - Stage 2
+
+    Refer: https://github.com/keras-team/keras-applications/blob/master/keras_applications/efficientnet.py
```

Comment:
Prefer providing the exact code block link.

Reply:
Cool